README.md

```diff
@@ -1 +1,137 @@
-# Recipe-app
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📰 Recipe App <a name="about-project"></a>
+
+> "Recipe-App" ia a web application that keeps track of all your recipes and ingredients. It allows you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- <a href="https://www.ruby-lang.org/es/">ROR</a>
+- <a href="https://www.postgresql.org/">Postgresql</a>
+
+## Key Features <a name="key-features"></a>
+
+-  **Registered users will be able to create new foods and recipes..**
+- **Users will be able to Add Food.**
+- **Users will be able to Add Recipe..**
+- **Users will authenticated before login..**
+- **Users will be able to View Food and Recipe..**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Command line or terminal
+- IDE such as VsCode.
+- Ruby and ROR needs to be installed on your system.
+
+### Setup
+
+Clone this repository to your desired folder:
+Example command:
+
+```sh
+  git clone https://github.com/huzaifakhan0308/Recipe-app.git
+  cd Recipe-app
+
+```
+
+### Install
+
+Example command:
+
+```sh
+  cd Recipe-app
+  bundle install (install all dependencies)
+
+```
```

Comment:
:red_circle: I think you did forget to include two important steps here before running the web application.  Please include the commands to create the database and run the migrations